### ISSUE TITLE
As soon as edgemesh is deployed, cloudcore will collapse.

### ISSUE THREAD

**Issue by gitlijian** [kubeedge/kubeedge#2459]
**What happened**:
As soon as edgemesh is deployed, cloudcore will collapse.
**What you expected to happen**:
Stability of edgemesh
**How to reproduce it (as minimally and precisely as possible)**:
Edgemesh is unstable and domain name resolution fails every few minutes
**Anything else we need to know?**:
The logs of cloudcore are as follows:
![image](https://user-images.githubusercontent.com/15320599/103061501-76a12900-45e6-11eb-9346-f92b96ed6a80.png)

**Environment**:
<details><summary>Kubernets Version</summary>
```console
$ kubelet --version
# paste output here
Kubernetes v1.19.3
$ kubectl version
# paste output here
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:41:49Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}

```

</details>

<details><summary>CloudCore Version</summary>

```console
$ cloudcore --version
# paste output here
KubeEdge v1.5.0
```

</details>

<details><summary>EdgeCore Version</summary>

```console
$ edgecore --version
# paste output here
KubeEdge v1.5.0

```

</details>

**CloudSide Environment**:
<details><summary>Hardware configuration</summary>

```console
$ lscpu
# paste output here
Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              56
On-line CPU(s) list: 0-55
Thread(s) per core:  2
Core(s) per socket:  14
Socket(s):           2
NUMA node(s):        2
Vendor ID:           GenuineIntel
CPU family:          6
Model:               63
Model name:          Intel(R) Xeon(R) CPU E5-2683 v3 @ 2.00GHz
Stepping:            2
CPU MHz:             1200.077
CPU max MHz:         3000.0000
CPU min MHz:         1200.0000
BogoMIPS:            4000.07
Virtualization:      VT-x
L1d cache:           32K
L1i cache:           32K
L2 cache:            256K
L3 cache:            35840K
NUMA node0 CPU(s):   0-13,28-41
NUMA node1 CPU(s):   14-27,42-55
Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault epb invpcid_single pti intel_ppin ssbd ibrs ibpb stibp tpr_shado
```

</details>

<details><summary>OS</summary>

```console
$ cat /etc/os-release
# paste output here
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

```
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

</details>

<details><summary>Kernel</summary>

```console
$ uname -a
# paste output here
Linux ndsl84 4.15.0-117-generic #118-Ubuntu SMP Fri Sep 4 20:02:41 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

```

</details>

<details><summary>Go version</summary>

```console
$ go version
no install go
# paste output here
```

</details>

<details><summary>Others</summary>
</details>

**EdgeSide Environment**:
<details><summary>Hardware configuration</summary>

```console
$ lscpu
# paste output here
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                32
On-line CPU(s) list:   0-31
Thread(s) per core:    2
Core(s) per socket:    8
Socket(s):             2
NUMA node(s):          2
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 63
Model name:            Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
Stepping:              2
CPU MHz:               2704.000
CPU max MHz:           3600.0000
CPU min MHz:           1200.0000
BogoMIPS:              6401.19
Virtualization:        VT-x
L1d cache:             32K
L1i cache:             32K
L2 cache:              256K
L3 cache:              20480K
NUMA node0 CPU(s):     0-7,16-23
NUMA node1 CPU(s):     8-15,24-31
Flags:                 fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm epb invpcid_single ssbd ibrs ibpb stibp kaiser tpr_shadow vnmi flexpriority ept vpid fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm xsaveopt cqm_llc cqm_occup_llc dtherm ida arat pln pts md_clear flush_l1d

```

</details>

<details><summary>OS</summary>

```console
$ cat /etc/os-release
# paste output here
NAME="Ubuntu"
VERSION="16.04.7 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.7 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial

```

</details>

<details><summary>Kernel</summary>

```console
$ uname -a
Linux ndsl86 4.4.0-190-generic #220-Ubuntu SMP Fri Aug 28 23:02:15 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

# paste output here
```

</details>

<details><summary>Go version</summary>

```console
$ go version
# paste output here
no install go
```

</details>

<details><summary>Others</summary>
</details>

**Comment by daixiang0** [comment]
EdgeMesh is a module you enable in EdgeCore, do you mean CloudCore crash when EdgeCore start EdgeMesh?

**Comment by threestoneliu** [comment]
PR [2387](https://github.com/kubeedge/kubeedge/pull/2387) may have solved this problem. You can build cloudcore from the master or release-1.5 branch and try it again.

**Comment by gitlijian** [comment]
yes





---Original---

**Comment by gitlijian** [comment]
部署一个edgemesh的时候。
我从云端部署一个服务 比如：
apiVersion: v1
kind: Service
metadata:
  name: cece-svc
  namespace: default
spec:
  clusterIP: None
  selector:
    app: rocketmq-cece-server
  ports:
    - name: http-0
      port: 8067
      protocol: TCP
      targetPort: 8067
然后cloudcore就挂掉了
我之前在Ubuntu18.04.4上部署之后没有出现过这种情况，不知道是不是因为边缘节点Ubuntu版本是16的原因？

**Comment by gitlijian** [comment]
这几点发现是，一部署svc，cloudcore 1.5就会崩；
但如果在启动cloudcore之前部署service，cloudcore就不会崩掉。

**Comment by daixiang0** [comment]
Try examples in https://kubeedge.io/en/docs/developer/edgemesh_guide/ first, if works, it would be related to your config issues.

**Comment by gitlijian** [comment]
今天把问题找到了，如果有kubeedge节点处于notready状态，部署svc的话， cloudcore就会崩掉，如果所有kubeedge节点都处于ready状态，就不会出现这种情况；
另外再说一个问题：
kubeedge会不断去拉flannel和kubeproxy，这是很占磁盘的；
![image](https://user-images.githubusercontent.com/15320599/103206766-8a76c300-4937-11eb-9a72-0cbba7392e98.png)

**Comment by gitlijian** [comment]
![image](https://user-images.githubusercontent.com/15320599/103206796-a1b5b080-4937-11eb-9055-546091af5ef1.png)
这是死掉的容器，这还只是一部分

**Comment by daixiang0** [comment]
It is fixed by #2035

**Comment by daixiang0** [comment]
Upgrade as [comment](https://github.com/kubeedge/kubeedge/issues/2459#issuecomment-750766488) said :)
